DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfigurationController.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem
+{
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Authorize(Policy = CustomPolicies.UserCentreAdminOnly)]
+    [Route("/TrackingSystem/CentreConfiguration")]
+    public class CentreConfigurationController : Controller
+    {
+        private readonly ICentresDataService centresDataService;
+
+        public CentreConfigurationController(ICentresDataService centresDataService)
+        {
+            this.centresDataService = centresDataService;
+        }
+
+        public IActionResult Index()
+        {
+            var centreId = User.GetCentreId();
+
+            var centreDetails = centresDataService.GetCentreDetailsById(centreId);
+
+            var model = new CentreConfigurationViewModel(centreDetails.CentreName, centreDetails.RegionName);
+
+            return View("~/Views/TrackingSystem/CentreConfiguration/Index.cshtml", model);
```

Comment:
Note that the view specified here is not just "Index" as it is not picked up automatically. This is because the controller name is "CentreConfigurationController" rather than "TrackingSystemController". 

The ticket states we will have a lot of controllers in the Tracking System so it is best to have a new folder for them. Assuming this, I created the controller as its own CentreConfigurationController, rather than making a partial TrackingSystemController as is done elsewhere (e.g. LearningPortal). I did this because it will make the individual controllers a little bit easier to manage and unit test, but a consequence is the automatic pathing doesn't work right so the views have to be specified manually. If we want to switch to using a single TrackingSystemController that uses partial classes for all its "child controllers" then it will be easy enough to change back. Thoughts?

Reply:
Had a quick google, and it might be possible to define custom 'search locations' in addition to the default ones. Might be something to consider if we can set up some standard way of naming all this stuff so that we can add some simple rule
https://docs.microsoft.com/en-us/aspnet/core/mvc/views/overview?view=aspnetcore-5.0#how-controllers-specify-views

Reply:
Good call. I have looked into that and made some changes in StartUp.cs that follow along with this stackoverflow answer: https://stackoverflow.com/a/59218969